SFS_config.m

```diff
@@ -468,17 +473,34 @@
 
 %% ===== References ======================================================
 %
-% Spors, Rabenstein, Ahrens - The Theory of Wave Field Synthesis Revisited, 124
-% AES Convention, Paper 7358, 2008. http://bit.ly/ZCvyQ6
+% Ahrens (2012) - "Analytic Methods of Sound Field Synthesis", Springer,
+% https://doi.org/10.1007/978-3-642-25743-8
+%
+% Ahrens, Spors (2010) - "Sound Field Reproduction Using Planar and Linear
+% Arrays of Loudspeakers", Transactions on Audio, Speech, and Language
+% Processing, vol. 18, no. 8, pp. 2038-2050,
+% https://doi.org/10.1109/TASL.2010.2041106
+%
+% Born, Wolf (1999) - "Principles of Optics", Cambridge University Press, 7th
```

Comment:
Maybe your version in Rostock is a reprinted version of the 7th edition. But I have to admit that there counting scheme is a little bit awkward. If you look at the [page of the publisher ](http://www.cambridge.org/catalogue/catalogue.asp?isbn=9780521642224), you see that the first release of the 7th edition was 1999, this can also be found in the [extract of the first pages](http://assets.cambridge.org/97805216/42224/sample/9780521642224ws.pdf), which by the way is a reprint from 2003.
I'm not sure what the official way is how to deal with this, but as on the publisher page the use 1999 under classification (left on their page) it is maybe ok to stay with the current entry?